### PR TITLE
Fetching 'SHOW META' info added to `yii\sphinx\Query`

### PR DESCRIPTION
**Pull request by klimov-paul**
Fetching 'SHOW META' info added to `yii\sphinx\Query`.
`ActiveDataProvider` updated allowing to get total count from meta information.

### CODE REVIEW

File: Query.php [hunk 1]
```diff
@@ -191,24 +198,53 @@ public function one($db = null)
...
-            $rows = $dataReader->readAll();
-            $rawFacets = [];
-            while ($dataReader->nextResult()) {
-                $rawFacets[] = $dataReader->readAll();
+        $command = $this->createCommand($db);
+        $dataReader = $command->query();
+        $rows = $this->populate($dataReader->readAll());
+
+        $facets = [];
+        if (!empty($this->facets)) {
```
Comment (samdark) [inline Query.php]: Don't really need this check since $this->facets is an array.

**Comment by samdark** [comment]
Overall looks good (haven't tested it).

**Comment by pahanini** [comment]
@klimov-paul  Looks great for me  

PS. Here is one tiny difference between \yii\data\ActiveDataProvider and yii\sphinx\ActiveDataProvider since this PR. It's  behavior of pagination regarding pages after last one. First provider shows last page, but second one shows empty page.

I use both providers to build rest API. 
-  \yii\data\ActiveDataProvider fetching api/object/?page=$very_big_int last page data will be shown
-  \yii\sphinx\ActiveDataProvider fetching api/object/?page=$very_big_int empty page will be shown

But it's not big deal I'm sure

**Comment by klimov-paul** [comment]
It is up to developer to decide if he wants to use 'meta' info for total count or not.
Still I suppose I need to add a note about this behavior. Thank you.

**Comment by klimov-paul** [comment]
Merged manually.

**Pull request merged**